test/integration/merge-conflict.test.js

```diff
@@ -1,7 +1,7 @@
 import fs from 'node:fs'
 import path from 'node:path'
 
-import { jest } from '@jest/globals'
+import { expect, jest } from '@jest/globals'
```

Comment:
What is the rationale for adding it? `expect` was used in other tests too.

Reply:
This was my editor autocompleting it.